### ISSUE TITLE
"phase data in seconds is assummed when no rate parameter is given"

### ISSUE THREAD

**Issue by garyemiller** [aewallin/allantools#67]
I have looked in the documentation, but can find no definition of 'phase data'.  What should it looks like?

For example, I have data from a TIC about a PPS input:

```
0.999999994060
0.999999994632
1.000000014844
0.999999994485
0.999999995617
0.999999995685
1.000000017018
```

Is that phase data?  Or should it be in degrees, radians, percentages, or?

**Comment by aewallin** [comment]
I agree that documentation should be written explaining the most common data input formats and measurement-setups used to obtain the data.
